src/common/fullstory.ts

```diff
@@ -32,8 +33,14 @@ function initStorage() {
     return;
```

Comment:
I think just referencing `localStorage` can throw an exception.  In fact, it's the above line that is showing up in sentry.  https://sentry.io/organizations/covidactnow/issues/2188103312/?project=5444052&referrer=slack

The recommended way to detect if storage is available is here: https://developer.mozilla.org/en-US/docs/Web/API/Web_Storage_API/Using_the_Web_Storage_API#testing_for_availability